### PR TITLE
presence: Use theme background color.

### PR DESCRIPTION
**Pull request by WesleyAC**
Fixes: #4734

### CODE REVIEW

**Comment by WesleyAC** [comment]
before:

Screenshot_20210508-032452

after:

Screenshot_20210508-032312

File: src/common/PresenceStatusIndicator.js [hunk 1]
```diff
@@ -67,11 +66,13 @@ function MaybeOpaqueBackgroundWrapper(
   |}>,
 ) {
   const { useOpaqueBackground, style, children } = props;
+  const context = useContext(ThemeContext);
```
Comment (chrisbobbe) [inline src/common/PresenceStatusIndicator.js:69]: tiny nit:

I've more often skipped defining `context`, and done something like this instead:

```js
const { backgroundColor } = useContext(ThemeContext);
```

Or I guess if that conflicts with something, give it a different name like

```js
const { backgroundColor: themeBackgroundColor } = useContext(ThemeContext);
```

What do you think?

**Review by chrisbobbe — APPROVED** [review]
LGTM, thanks!

File: src/common/PresenceStatusIndicator.js (hunk 1 above)
Comment (WesleyAC) [reply src/common/PresenceStatusIndicator.js]: Good call, done :)

**Pull request merged**